ios/PolyPodApp/PodApi/PolyIn/Quads/PolyIn+Add.swift

```diff
@@ -16,10 +16,10 @@ extension PolyIn {
         
         do {
             try managedContext.save()
-            completionHandler(true)
+            completionHandler(nil)
```

Comment:
Er. Substituing a truish value by a falsish value...

Reply:
... which is anyway irrelevant... Maybe just leave it empty? `Optional`s allow this, right?

Reply:
Hm, maybe it could be left empty. It follows the overall pattern here (which is quite common in iOS APIs -> An error value of `nil` indicates success. There's no concept of falsiness in Swift.